src/test/resources/jenkins/metrics/configuration-as-code.yaml

```diff
@@ -0,0 +1,10 @@
+unclassified:
+  metricsaccesskey:
+    accessKeys:
+      - key:            "tDdG5Vsv-2-WDdHfI3QFPiU9-hcvKmWd2HL4CfVIFvUumQzz3qf6c0qt_HU4_lUh"
```

Comment:
There were some issues with handling arrays in JCasC. I recommend having a test with takes several items

Reply:
Are you fine if we finish this PR in the current state and I do that in a separate PR? Thanks

Reply:
Or did you mean that https://github.com/jenkinsci/metrics-plugin/pull/46/commits/992748c7502375239ac60fb203a9525ded7c0917?

Reply:
Yeah, that